apps/browser/src/background/runtime.background.ts

```diff
@@ -93,6 +93,10 @@ export default class RuntimeBackground {
   async processMessage(msg: any, sender: chrome.runtime.MessageSender) {
     switch (msg.command) {
       case "loggedIn":
+        // Send extension install message for vault onboarding component
+        // eslint disabled, do not want break statement here
+        await this.sendBwInstalledMessageToVault();
+      // eslint-disable-next-line
       case "unlocked": {
         let item: LockedVaultPendingNotificationsData;
```

Comment:
```suggestion
    switch (msg.command) {
      case "loggedIn": 
      case "unlocked": {
        let item: LockedVaultPendingNotificationsData;
        
        if (msg.command === "loggedIn") {
          await this.sendBwInstalledMessageToVault();
        }      
```

Hmmm, this whole thing is definitely messier than we'd like, and even before your added changes.

It might make sense instead to incorporate a conditional like this, which avoid usage of `eslint-disable-next-line`. It kind of stinks that we have to do a second conditional check, but that likely won't have a huge performance impact overall.